lib/fastly-rails/mongoid/surrogate_key.rb

```diff
@@ -35,8 +35,8 @@ def purge_all
         self.class.purge_all
       end
 
-      def service_id
-        self.class.service_id
+      def fastly_service_identifier
+        self.class.fastly_service_identifier
```

Comment:
I believe that `@@fastly_service_identifier` does not work here but if it does it would read a lot better.